src/components/ActionButton.tsx

```diff
@@ -39,13 +40,16 @@ export const ActionButton: React.FunctionComponent<IActionButtonProps> = (
 ) => {
   const { disabled, className, title, onClick, icon } = props;
   return (
-    <button
-      disabled={disabled}
-      className={classes(actionButtonStyle, className)}
-      title={title}
-      onClick={onClick}
-    >
-      <icon.react elementPosition="center" tag="span" />
-    </button>
+    <>
+      <Tooltip anchor={title}>{title}</Tooltip>
+      <Button
+        id={title}
+        className={classes(actionButtonStyle, className)}
+        disabled={disabled}
+        onClick={onClick}
+      >
+        <icon.react elementPosition="center" tag="span" />
+      </Button>
+    </>
```

Comment:
I would simply keep the title without introducing a tooltip

```suggestion
    <Button
      title={title}
      className={classes(actionButtonStyle, className)}
      disabled={disabled}
      onClick={onClick}
    >
      <icon.react elementPosition="center" tag="span" />
    </Button>
```